include/boost/geometry/extensions/gis/io/wkb/detail/writer.hpp

```diff
@@ -178,14 +180,16 @@ namespace detail { namespace wkb
 
             interior_rings_type interior_rings = boost::geometry::interior_rings(polygon);
```

Comment:
It seems here that interior rings are copied.
Is that a good idea? I think we should avoid the copying. We have already faced similar issues in the library and there is a sort-of standard way for addressing this.